TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+# CesiumJS and Webpack
+
+[Webpack](https://webpack.js.org/) is a popular and powerful tool for bundling JavaScript modules. It allows developers to structure their code and assets in an intuitive way and to load different kinds of files as needed with simple require statements. When building, it will trace code dependencies and pack these modules into one or more bundles that are loaded by the web browser.
+
```

Comment:
In our website tutorials, we've built the "You'll learn how to" as a bulleted list. I would recommend that here too. The paragraph below is good, but it is good to summarize it to the user as well.